projects/sbb-esta/angular-business/chip/src/chip-input/chip-input.component.ts

```diff
@@ -0,0 +1,261 @@
+    }
+  }
+  private _value: string[] = [];
+
+  /**
+   * Implemented as part of FormFieldControl.
+   * @docs-private
+   */
+  get inputId() {
+    return this.id;
+  }
+
+  /** Whether the chip-input has a value. */
+  get empty(): boolean {
+    return !this.selectionModel || this.selectionModel.isEmpty();
+  }
+
+  /**
+   * Event that emits whenever the raw value of the chip-input changes. This is here primarily
+   * to facilitate the two-way binding for the `value` input.
+   * @docs-private
+   */
+  @Output() readonly valueChange: EventEmitter<any> = new EventEmitter<any>();
+
+  focused = false;
+  errorState: boolean;
+  inputModel = '';
+  origin = new AutocompleteOriginDirective(this._elementRef);
+  selectionModel: SelectionModel<string>;
+
+  /** Whether filling out the chip-input is required in the form. */
+  private _required = false;
+
+  /** The aria-describedby attribute on the chip-input for improved a11y. */
+  private _ariaDescribedby: string;
+
+  /** Unique id for this input. */
+  private _uid = `sbb-chip-input-${nextUniqueId++}`;
+
+  /** Emits when the state of the option changes and any parents have to be notified. */
+  readonly stateChanges = new Subject<void>();
+
+  private _onTouchedCallback: () => void = () => {};
+  private _onChangeCallback: (_: any) => void = () => {};
+
+  constructor(
+    @Self() @Optional() public ngControl: NgControl,
+    private _elementRef: ElementRef,
+    private _changeDetectorRef: ChangeDetectorRef
+  ) {
+    this.selectionModel = new SelectionModel<string>(true);
+    if (this.ngControl) {
+      // Note: we provide the value accessor through here, instead of
+      // the `providers` to avoid running into a circular import.
+      this.ngControl.valueAccessor = this;
+    }
+
+    // Force setter to be called in case id was not specified.
+    this.id = this.id;
+  }
+
+  ngOnInit(): void {
+    if (this.autocomplete) {
+      this.autocomplete.optionSelected.subscribe(event => this.onSelect(event.option.value));
+    }
+  }
+
+  /**
+   * Sets the chip input's value. Part of the ControlValueAccessor interface
+   * required to integrate with Angular's core forms API.
+   *
+   * @param value New value to be written to the model.
+   */
+  writeValue(value: string[]): void {
+    this.selectionModel.clear();
+    if (value) {
+      value.forEach(v => this.selectionModel.select(v));
+    }
+    this._propagateChanges();
+  }
+
+  /**
+   * Disables the chip input. Part of the ControlValueAccessor interface required
+   * to integrate with Angular's core forms API.
+   *
+   * @param isDisabled Sets whether the component is disabled.
+   */
+  setDisabledState(isDisabled: boolean): void {
+    this.disabled = isDisabled;
+    this._changeDetectorRef.markForCheck();
+    this.stateChanges.next();
+  }
+
+  /**
+   * Adds a given value to the current selected values.
+   */
+  onSelect(option: string) {
+    if (!this.selectionModel.isSelected(option)) {
```

Comment:
I think the empty entry problem can be solved by checking if option is empty here.

```suggestion
    if (!option) {
      return;
    } else if (!this.selectionModel.isSelected(option)) {
```

Reply:
This would work if it was only the empty option being added - locally I also found that the typed text gets added when selecting an autocomplete option with enter. Must be a timing difference on whether I already emptied the input field or not. 
It does make make sense, as the keydown event fires for both the autocomplete component and the input element. That's why I'm trying to stop the enter event of the input field when an autocomplete option is active.